README.md

```diff
@@ -11,10 +11,21 @@ Ansible role to configure backups using [autorestic](https://autorestic.vercel.a
 
 None.
 
+## Assumptions
+
+Places versioned executables into `/opt/autorestic/bin` and `/opt/restic/bin` links these into `/usr/local/bin`.
```

Comment:
Maybe there should be two role variables for this? Something like `autorestic_install_directory` and `autorestic_restic_install_directory`? Where the defaults are `/usr/local/bin` but can obviously be overridden to whatever the user desires, this would work for your use-case right?


where the user can specify where to install the autorestic and restic binaries to?

This way, we can set the default to `/usr/local/bin`

Something like `autorestic_install_directory` maybe?

Reply:
how about we define
autorestic_install_dir: /opt/autorestic
autorestic_restic_install_dir: /opt/restic
autorestic_binary_dir: /usr/local/bin
autorestic_restic_binary_dir: /usr/local/bin

or is that too much variety?
come to think of it, naming it is not so trivial it seems. is the install dir the final location of a binary/link to it? that might be a bit overthought.

Reply:
oh i missed this one. done now.